bufr/src/main/java/ucar/nc2/iosp/bufr/tables/TableC.java

```diff
@@ -20,6 +20,7 @@
     tableCdesc[4] = "add associated field";
     tableCdesc[5] = "signify character";
     tableCdesc[6] = "signify data width for next descriptor";
+    tableCdesc[7] = "increase scale, reference value, and data width";
```

Comment:
did you find what this means? I dont see an implementation in DataDescriptorTreeConstructor??

Reply:
It was in there already, just not defined in `TableC.java`

https://github.com/Unidata/netcdf-java/blob/90ae49b86ba094706af504401f1abe0c7db1f418/bufr/src/main/java/ucar/nc2/iosp/bufr/DataDescriptorTreeConstructor.java#L330

Once `changeWtf` get's set, it is applied here until it gets turned off:

https://github.com/Unidata/netcdf-java/blob/90ae49b86ba094706af504401f1abe0c7db1f418/bufr/src/main/java/ucar/nc2/iosp/bufr/DataDescriptorTreeConstructor.java#L410-L423